src/vm.rs

```diff
@@ -149,6 +151,8 @@ impl<VCpuType: VirtualCPU> UhyveVm<VCpuType> {
 			"gdbstub is only supported with one CPU"
 		);
 
+		let file_map = params.mount.as_deref().and_then(UhyveFileMap::new);
```

Comment:
Reads a bit more complicated than `UhyveFileMap::new(&params.mount)` if you ask me.

Reply:
> Reads a bit more complicated than UhyveFileMap::new(&params.mount) if you ask me.

Perhaps. But, if the specific parameter is missing, this helps us spare the function call as well as checking whether the parameter is none within `UhyveFileMap::new` itself, as it just straight up instantly returns `None` in that case with less lines of code.